### DISCUSSION TITLE
Change in email send behavior on package publish? Or related to granular token?

### DISCUSSION THREAD

**Discussion by janpio** [npm/feedback#1030] [Registry] [answered]
We just noticed that our bot email account, which is connected to the bot Npm account we use to publish our packages, stopped receiving emails to confirm the publish of a package in July. We are trying to understand what triggered this change in behavior to make sure it is intentional and nothing is broken on our side.

Around the time this change happened, we might have switched from an old-style to a new granular token to publish. Could that be connected to this change?

Unfortunately I could find no documentation about the emails that are expected. I could only find this discussion here that suggest being able to disable these emails: https://github.com/npm/feedback/discussions/424

**Comment by tiagonbotelho** [comment, marked as answer]
Hi @janpio, I'm sorry you are encountering issues with our package publish emails not being delivered to you.

However, I wasn't able to reproduce the issue myself, so for us to troubleshoot further it'll be helpful to get a bit more information. If possible, I'd suggest opening a ticket with our team in https://www.npmjs.com/support and we'll be sure to dig further into this for you 😊
